heliopy/util/config.py

```diff
@@ -22,11 +23,11 @@ def get_config_file():
     config_filename = 'heliopyrc'
 
     # Get user configuration location
-    home_dir = os.path.expanduser("~")
-    config_file_1 = os.path.join(home_dir, '.heliopy', config_filename)
+    home_dir = Path(os.path.expanduser("~"))
```

Comment:
This can actually just be `pathlib.Path.home()` (https://docs.python.org/3/library/pathlib.html#pathlib.Path.home)